### ISSUE TITLE
Test failures

### ISSUE THREAD

**Issue by FRidh** [wireservice/csvkit#1041]
Building and testing with Python 3.7.4:
```
======================================================================
FAIL: test_linenumbers (tests.test_utilities.test_sql2csv.TestSQL2CSV)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/csvkit-1.0.4/tests/test_utilities/test_sql2csv.py", line 140, in test_linenumbers
    self.assertTrue('1,1,2,3' in csv)
AssertionError: False is not true

======================================================================
FAIL: test_no_header_row (tests.test_utilities.test_sql2csv.TestSQL2CSV)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/csvkit-1.0.4/tests/test_utilities/test_sql2csv.py", line 133, in test_no_header_row
    self.assertTrue('1,2,3' in csv)
AssertionError: False is not true

======================================================================
FAIL: test_unicode (tests.test_utilities.test_sql2csv.TestSQL2CSV)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/csvkit-1.0.4/tests/test_utilities/test_sql2csv.py", line 126, in test_unicode
    self.assertEqual(csv.strip(), expected)
AssertionError: 'foo,bar,baz\n1.0,2.0,3\n4.0,5.0,ʤ' != 'foo,bar,baz\n1,2,3\n4,5,ʤ'
  foo,bar,baz
- 1.0,2.0,3
?  --  --
+ 1,2,3
- 4.0,5.0,ʤ+ 4,5,ʤ

----------------------------------------------------------------------
Ran 236 tests in 20.347s

FAILED (failures=3)
builder for '/nix/store/a6qzzbxs6p6n229178b4m8br88fc4cxj-csvkit-1.0.4.drv' failed with exit code 1
```

**Comment by jpmckinney** [comment]
Duplicate and fixed #1027.

**Comment by FRidh** [comment]
Great, thanks!
